base/float.jl

```diff
@@ -757,11 +779,9 @@ for fn in (:float,)
     @eval begin
```

Comment:
@Sacha0 `float` has a reprieve from deprecation, right? so may as well de-eval and de-loop this to just be over `float` since it's the last function left here?